sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -1292,6 +1292,23 @@ esimd_cbit(T src) {
   return Result[0];
 }
 
+/// Scalar version of \c esimd_cbit.
```

Comment:
Please reformulate. The above function say is it is 'Scalar version of \c esimd_cbit' too. Maybe add comment in which cases this one is invoked.

Reply:
Done.